torch/fx/graph.py

```diff
@@ -480,6 +499,19 @@ def get_attr(self, qualified_name: str, type_expr: Optional[Any] = None) -> Node
             The same insertion point and type expression rules apply for this method
             as ``Graph.create_node``.
         """
+        if (self.owning_module and 
+                self.owning_module.get_submodule(qualified_name) is None
+                and self.owning_module.get_parameter(qualified_name) is None
+                and self.owning_module.get_buffer(qualified_name) is None):
```

Comment:
I'm a bit worried about the perf implications of all these checks: internally they're redundantly doing the same traversal of the module hierarchy. Maybe that part can be split off into a `get_attr` on `nn.Module` which these other APIs call and which we can call here